modules/acid/src/main/ts/ephox/acid/gui/ColourPicker.ts

```diff
@@ -72,20 +97,37 @@ const makeFactory = (
       return (form: AlloyComponent, simulatedEvent: SimulatedEvent<ColourEvents.PaletteUpdateEvent>) => {
         const value = simulatedEvent.event.value;
         const oldRgb = state.paletteRgba.get();
-        const hsvColour = HsvColour.fromRgb(oldRgb);
-        const newHsvColour = HsvColour.hsvColour(hsvColour.hue, value.x, (100 - value.y));
-        const rgb = RgbaColour.fromHsv(newHsvColour);
-        const nuHex = HexColour.fromRgba(rgb);
-        runUpdates(form, nuHex, updates);
+        const oldHex = HexColour.fromRgba(oldRgb);
+        const oldHue = state.paletteHue.get();
+        const newHsv = HsvColour.hsvColour(oldHue, value.x, (100 - value.y));
+        const newHex = Transformations.hsv2hex(newHsv);
+
+        if (oldHex.value !== newHex.value) {
```

Comment:
Is this still needed? as this didn't work properly anyways and we solved the case it was added for by not firing the pallete `onChange` callback.